src/constants/v1/index.ts

```diff
@@ -8,7 +8,8 @@ const V1_FACTORY_ADDRESSES: { [chainId in ChainId]: string } = {
   [ChainId.ROPSTEN]: '0x9c83dCE8CA20E9aAF9D3efc003b2ea62aBC08351',
   [ChainId.RINKEBY]: '0xf5D915570BC477f9B8D6C0E980aA81757A3AaC36',
   [ChainId.GÖRLI]: '0x6Ce570d02D73d4c384b46135E87f8C592A8c86dA',
-  [ChainId.KOVAN]: '0xD3E51Ef092B2845f10401a0159B2B96e8B6c3D30'
+  [ChainId.KOVAN]: '0xD3E51Ef092B2845f10401a0159B2B96e8B6c3D30',
+  [ChainId.XDAI]: '0x70E969444603980c94f9fCF612F00767744F92E7'
```

Comment:
Do we even care about v1 anymore? Wondering if we should eventually remove this/all v1 code and save us the hassle

Reply:
We don't care, although is the same as above, typescript ask for it, and is nice to add the right value.